ckanharvester.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+
+from sqlalchemy import create_engine
+from sqlalchemy import MetaData
+from ckanext.harvest.model import HarvestObject
+import logging
+from base import HarvesterBase
+
+log = logging.getLogger(__name__)
+
+
+class CKANHarvester(HarvesterBase):
+    def _connect(self, user, password, db, host='192.168.56.101'):
+        url = 'mysql://{}:{}@{}/{}?charset=utf8'
+        url = url.format(user, password, host, db)
+        con = create_engine(url)
+        meta = MetaData(bind=con, reflect=True)
+        return con, meta
+
+    def info(self):
+        return {
+            'name': 'opendatagov',
+            'title': 'opendata.gov.lt',
+            'description': 'Harvest opendata.gov.lt',
+            'form_config_interface': 'Text'
+        }
+
+    def gather_stage(self, harvest_job):
+        log.debug('In opendatagov gather_stage')
+        con, meta = self._connect('root', 'root', 'opendatagov')
+        results = meta.tables['t_rinkmena']
```

Comment:
Name `results` is really misleading, this is a table and should be called table, for example `t_rinkmena` or `rinkmena`. And why you didn't liked [previous approach](https://github.com/ivpk/opendata.gov.lt-mysql-import/blob/master/odgovltimport.py#L134):

```python
class Tables(object):
    user = db.tables['t_user']
    istaiga = db.tables['t_istaiga']
    rinkmena = db.tables['t_rinkmena']
    rinkmenu_logas = db.tables['t_rinkmenu_logas']

self.t = Tables
```